src/main/java/seedu/address/logic/parser/FindPropertyCommandParser.java

```diff
@@ -28,7 +34,24 @@ public FindPropertyCommand parse(String args) throws ParseException {
 
         String[] nameKeywords = trimmedArgs.split("\\s+");
 
-        return new FindPropertyCommand(new PropertyContainsKeywordsPredicate(Arrays.asList(nameKeywords)));
+        List<Predicate<Property>> predicates = new ArrayList<>();
+        List<String> keywords = new ArrayList<>();
+
+        for (String s : nameKeywords) {
+            if (s.startsWith(String.valueOf(PREFIX_PROPERTY_PRICE_MORE))) {
+                predicates.add(new PropertyPricePredicate(s.split("/")[1], false));
+            } else if (s.startsWith(String.valueOf(PREFIX_PROPERTY_PRICE_LESS))) {
+                predicates.add(new PropertyPricePredicate(s.split("/")[1], true));
+            } else {
+                keywords.add(s);
+            }
+        }
```

Comment:
Just a suggestion: Would you like to consider using the `ArgumentTokenizer` class here?

Reply:
The keywords have no prefixes though, so I think a for loop like this would be more convenient.